flink-core/src/main/java/org/apache/flink/api/common/io/BinaryInputFormat.java

```diff
@@ -337,4 +402,18 @@ public int read(byte[] b, int off, int len) throws IOException {
 			return totalRead;
 		}
 	}
+
+	// ============================================================================================
```

Comment:
In most parts of the code we have dashes instead of equals signs, I think, like this:
```
// ------------------------------------------------------------------------
//  Checkpointing
// ------------------------------------------------------------------------
```